src/components/canrawview/canrawview_p.h

```diff
@@ -58,12 +61,15 @@ class CanRawViewPrivate : public QWidget {
 
         QList<QVariant> qvList;
         QList<QStandardItem*> list;
+    
+        int frameID = frame.frameId();
+        QString time = QString::number((double)timer->elapsed() / 1000, 'f', 2);        
```

Comment:
`static_cast<double>(...)` and `1000.0` to explicitly denote the type of the value.